odl/util/phantom.py

```diff
@@ -538,6 +538,92 @@ def rect(x):
     out = discr.element(ellipse)
     out += discr.element(rect)
     return out.ufunc.minimum(1, out=out)
+    
+    
+def disc_phantom(discr, smooth=True, taper=20.0):
```

Comment:
What's the motivation for this phantom? To me it seems very similar to the submarine guy.